src/utils/klage/klageUtils.ts

```diff
@@ -128,26 +128,48 @@ export const erKlageFerdigbehandlet = (klage: Klage): boolean => {
     return erKlageIverksattAvvist(klage) || erOversendtKlageFerdigbehandlet(klage) || erKlageOmgjortFerdigstilt(klage);
 };
 
-export const erKlageOmgjortFerdigstilt = (k: Klage): k is FerdigstiltOmgjortKlage =>
-    k.status === KlageStatus.FERDIGSTILT_OMGJORT;
+export const erKlageOmgjortFerdigstilt = (
```

Comment:
TODO: ikke helt fornøyd med denne da man ikke vet hvilken man skal aksessere, alternativet er to sjekker for en og samme ting selvom de blir levert på ulike paths men back kan jo returnere på samme men det blir misvisende såe ts skjønte ikke typene her uansett så det beste hadde kanskje vært å returnert et object som er det ene som finnes så det er lett å jobbe med